dev/src/TestAzureAccount.ts

```diff
@@ -56,7 +57,7 @@ export class TestAzureAccount implements AzureAccount, types.TestAzureAccount {
             environment: credentials.environment,
             userId: '',
             tenantId: tenantId,
-            credentials: credentials
+            credentials2: credentials
```

Comment:
They changed the property?  Is this to have backwards compatibility?  Is there any concern that they'll change this back to just credentials after support for the old SDKs are expired?

Reply:
Yes it is to maintain backwards compatibility, and no I'm not worried about them changing the name

https://github.com/microsoft/vscode-azure-account/issues/140#issuecomment-647969090